http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -644,7 +644,8 @@ object Boot extends IOApp {
         googleDiskService,
         appDescriptorDAO,
         nodepoolLock,
-        googleDependencies.googleResourceService
+        googleDependencies.googleResourceService,
+        googleDependencies.googleComputeService
```

Comment:
We need this so I can query the memory and gpu based on the machine type for the Galaxy app :'(

Reply:
Hmm.. is this needed because the machineType is in the DB but not the gpu information?

Reply:
Indeed :(